src/openforms/authentication/contrib/yivi_oidc/tests/test_admin.py

```diff
@@ -0,0 +1,268 @@
+import json
+
+from django.urls import reverse_lazy
+
+from django_webtest import WebTest
+from maykin_2fa.test import disable_admin_mfa
+
+from openforms.accounts.tests.factories import UserFactory
+from openforms.authentication.tests.factories import AttributeGroupFactory
+
+from ..models import AttributeGroup
+
+
+@disable_admin_mfa()
+class AttributeGroupAdminTests(WebTest):
+    admin_export_url = reverse_lazy("admin:yivi_oidc_attributegroup_export")
+    admin_import_url = reverse_lazy("admin:yivi_oidc_attributegroup_import")
+
+    def _get_value_for_select_field(self, field, file_type):
+        json_option = next((t for t in field.options if file_type in t), None)
+        assert json_option is not None
+
+        return json_option[0]
+
+    def test_successful_export_attribute_groups(self):
+        user = UserFactory.create(
+            is_staff=True,
+        )
+
+        # Create test data
+        AttributeGroupFactory(
+            name="group1",
+            uuid="12345678-abcd-1234-abcd-1234567890ab",
+            description="For retrieving group 1 data",
+            attributes=["foo", "bar", "foo.bar"],
+        )
+        AttributeGroupFactory(
+            name="group2",
+            uuid="12345678-abcd-1234-abcd-1234567890cd",
+            description="For retrieving group 2 data",
+            attributes=["zoo", "zar"],
+        )
+
+        # Go to export page
+        export_page = self.app.get(self.admin_export_url, user=user)
+        form = export_page.forms["export-attributegroups"]
+
+        # Select json file format, and export
+        form["file_format"] = self._get_value_for_select_field(
+            form["file_format"], "json"
+        )
+        response = form.submit()
+
+        # Expect that the export was successful
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content_type, "application/json")
+        self.assertGreater(int(response["Content-Length"]), 0)
+
+        # Expect that the export contains all attribute groups
+        decoded_response = json.loads(response.body.decode())
```

Comment:
```suggestion
        decoded_response = response.json()
```
(django-)webtest has this covered :)